api/src/main/java/org/openmrs/module/referenceapplication/builtinreports/reports/ListOfDiagnosis.java

```diff
@@ -0,0 +1,92 @@
+package org.openmrs.module.referenceapplication.builtinreports.reports;
+
+import org.openmrs.module.reporting.ReportingConstants;
+import org.openmrs.module.reporting.dataset.definition.SqlDataSetDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.openmrs.module.reporting.report.manager.BaseReportManager;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Jude on 6/3/2017.
+ */
```

Comment:
We do not include Created by... in comments by convention. Please remove it from all newly created classes. No worries, your contribution is visible in the git history.